src/shared/Core/Trace2.cs

```diff
@@ -1,7 +1,9 @@
 using System;
 using System.Collections.Generic;
+using System.ComponentModel;
 using System.IO;
 using System.IO.Pipes;
+using System.Reflection.Emit;
```

Comment:
Is this used? I don't see any changes here that require reflection. Perhaps I'm missing something though.